internal/extproc/processor.go

```diff
@@ -184,8 +204,13 @@ func (p *Processor) ProcessResponseBody(_ context.Context, body *extprocv3.HttpB
 		return &extprocv3.ProcessingResponse{Response: &extprocv3.ProcessingResponse_ResponseBody{}}, nil
 	}
 
-	headerMutation, bodyMutation, tokenUsage, err := p.translator.ResponseBody(p.responseHeaders, br, body.EndOfStream)
+	headerMutation, bodyMutation, tokenUsage, err := p.translator.ResponseBody(p.responseHeaders, br, body.EndOfStream, p.backendName, p.modelName)
 	if err != nil {
+		if p.backendName != "" && p.modelName != "" {
```

Comment:
We probably should not have this check here, if the model name or backend are not there we should fail upfront

Reply:
removed